library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_forms.scss

```diff
@@ -246,7 +246,8 @@ input[type="search"] {
 }
 
 .radio + .radio,
-.checkbox + .checkbox {
+.checkbox + .checkbox,
+.reduction-top {
```

Comment:
I'll try avoid margin-top negatives...
Could we reduce the margin-bottom from the top element?

Reply:
I don't know, i will try now

Reply:
Yes, it works. In some minutes, i will send the new "version" with that changes

Reply:
Same, try to do not modify the styles here!